portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -55,6 +67,9 @@ public long getTimestamp() {
      * @return A string containing the comment's contents.
      */
     public String toString() {
-        return id + ": " + text + " " + timestamp;
+        return "ID: " + id + "," +
+               "Nickname: " + nickname + "," +
+               "Text: " + text + "," +
+               "IsOwner: " + isOwner;
```

Comment:
Was this tested to ensure newline/spaces were properly placed?

Reply:
I haven't tested it but same with the get functions, I may just remove this since I don't use it and don't plan to.

Reply:
Removed the toString function and resolving this.